addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -1842,6 +1842,22 @@ QUnit.test('remove an uploading attachment aborts upload', async function (asser
     });
 });
 
+QUnit.test('basic rendering when thread has no name', async function (assert) {
+    assert.expect(1);
+
+    await this.start();
+    const thread = this.env.models['mail.thread'].create({
+        composer: [['create', { isLog: false }]],
+        id: 20,
+        model: 'res.partner',
+    });
+    await this.createComposerComponent(thread.composer, {hasFollowers: true});
+    assert.strictEqual(
+        document.querySelector('.o_Composer_followers b:not(.text-muted)').innerHTML,
+        " this document "
+    );
```

Comment:
This selector is too specific to the way it's implemented. Instead, assert on `. o_Composer_followers`'s `textContent`, which is something like _"To: Followers of this document"_.